Independent container updates (#1080)
freedomofpress/dangerzone: docs/developer/independent-container-updates.md

# Independent Container Updates

> Since version 0.10.0, Dangerzone is able to ship container images independently

Author: name it "the sandbox" somewhere.

from releases of the software.

> One of the main benefits of doing so is to shorten the time neede to distribute the security fixes for the containers. Being the place where the actual conversion of documents happen, it's a way to keep dangerzone users secure.

Author: ```suggestion
The main benefit is to shorten the time needed to distribute sandbox security fixes.
```

> If you are a dangerzone user, this all happens behind the curtain, and you should not have to know anything about that to enjoy these "in-app" updates. If you are using dangerzone in an air-gapped environment, check the sections below.

Author: That makes me think that we should refer to the container image as "the sandbox" everywhere in the app. Wording changes are needed.
